package.json

```diff
@@ -130,6 +130,7 @@
     "jest-cli": "24.8.0",
     "jest-dom": "3.1.4",
     "jest-styled-components": "6.3.1",
+    "jest-watch-typeahead": "^0.3.1",
```

Comment:
Please pin the version number by removing the ^ (and rerun `npm i` for a small package-lock update).

Reply:
Whoops, done. `package-lock.json` stayed the same.